tools/stress-cluster/chaos/examples/stress-debug-share-example/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: stress-test-addons
-  repository: https://azuresdkartifacts.z5.web.core.windows.net/stress/
+  repository: file:///home/ben/azs/azure-sdk-tools/tools/stress-cluster/cluster/kubernetes/stress-test-addons
```

Comment:
I assume you don't want a local drive here.

Reply:
But let's consider, for a moment, if we can accept this as a new standard.

Reply:
@richardpark-msft with the new update, everyone will have to mount my local drive

Reply:
Awesome that means we won't need to host an insecure storage container :)

Reply:
We're onto something here.